QuestionBot/QuestionBot/Model/MessageEmitter.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace QuestionBot.Model {
+    public class MessageEmitter : IMessageEmitter {
+        private string _lineInput = "";
+        private static IList<IMessageListener> _listeners = new List<IMessageListener>();
+
+        public void Start() {
+            while( _lineInput != "/exitQuestionbot" ) {
```

Comment:
How come you added an exit command? (Not saying this is a bad idea - in fact, it could be a good one - just wondering your thoughts)